clients/fossid-webapp/build.gradle.kts

```diff
@@ -17,7 +17,9 @@
  * License-Filename: LICENSE
  */
 
+val jacksonVersion: String by project
```

Comment:
Please sort these alphabetically.

Reply:
the different vals are already sorted or do you mean something else ?

Reply:
Sorry, you're right, my mistake. Please ignore my comment.